### DISCUSSION TITLE
The "todo" status should really be "to do" in this context.

### DISCUSSION THREAD

**Discussion by jdshkolnik** [community/community#4933] [Projects and Issues]
It's like "setup vs. set up". You're not saying the issue is "a todo work type", you're saying this "is work to do" just like you'd be saying it's "work in progress" or "work is done".

**Comment by iamrajee** [comment]
++

**Reply by jdshkolnik** [reply]
Thanks. It'd also help to click the little voting arrow above the counter to the left of the initial post since that bumps the rank.
